hostscripts/rpm-packaging/createproject.py

```diff
@@ -82,10 +82,10 @@ def upload_meta(project, build_repository, linkproject):
                   'projectlink': projectlink,
                   'build_repository': build_repository})
 
-    with tempfile.NamedTemporaryFile(delete=False) as meta:
+    with tempfile.NamedTemporaryFile() as meta:
         meta.write(templ)
+        meta.file.flush()
```

Comment:
why file.flush and not flush() ? according to documentation NamedTemporaryFile returns a file-like object so calling flush on that one seems more correct to me (I haven't tested it)